train-A-app/src/app/app.config.ts

```diff
@@ -25,9 +33,13 @@ export const appConfig: ApplicationConfig = {
     provideStore({ roleState: rolesReducer.reducer }),
     provideState(stationFeature),
     provideState(carriageFeature),
+    provideState(searchFeature),
+    provideEffects([CarriageEffects]),
+    provideEffects([TripEffects]),
```

Comment:
We have a little mess here. All effects should be in one array.

Reply:
I added it to PR.